marxs/optics/multiLayerMirror.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+from astropy.io import ascii
+
+from .base import FlatOpticalElement
+from ..math.pluecker import *
+
+
+class MultiLayerMirror(FlatOpticalElement):
+    '''Multilayer mirror with varying layer thickness along one axis
+    
+    The distance between layers changes along the local y axis.
+    All reflectivity data is assumed to be for a single, desired angle. There
+    is currently no way to enter varying reflection that depends on the angle
+    of incidence.
+    Provide reflectivity data in a file with columns:
+    	'X(mm)' - position along the "changing" axis
+    	'Peak lambda' - wavelength with maximum reflection at a given position
+    	'Peak' - maximum reflection at a given position
+    	'FWHM(nm)' - full width half max, measure of width of reflection Gaussian peaks
+    
+    Parameters
+    ----------
+    fileCode: string
+    	serial number of mirror, used to open reflection data file
+    '''
+    def __init__(self, fileCode, **kwargs):
+        self.fileName = './marxs/optics/docs/' + fileCode + '.txt'
```

Comment:
This is fine for now, but it might be more general to allow users to put in the full path name and file name. They way this is written has two potential problems in the future:
- All these optical files have to be installed with the "marxs" package. Imagine Herman uses a different mirror next year. Would he `git clone`, add the data files, submit a pull request, etc.? Ten years from now, the git repository will be cluttered with mirror files for mirrors that have long been broken.
- This requires the user to run python fro the directory `marxs`. That's convenient for development, but if `marxs` is properly installed on your machine, then is should really work from any location.

I suggest to move the data files to a new directory `data` or maybe `examples/data` and allow the user to pass in a full path. (We can do other tricks like "It it's a full path, use that, if it's not found, use the version in the `data` directory" later, but for now that's probably a waste of time and path + filename will just work fine.) Although I argued above against having all data for all experiments in the git repro, it's good to have it for one mirror as an example of the format and for the tests.